ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/governedParticipantsServiceAgent.service.ts

```diff
@@ -18,8 +18,11 @@ export class GovernedParticipantsServiceAgent extends OperatorServiceAgent<any>
         super(session, configuration, http, logger);
     }
 
-    public add(alias: string): Observable<void> {
-        return this.postObservable(`/participants`, { alias });
+    public addParticipant(
+      studyGuid: string,
+      body: Record<string, any> = {},
+    ): Observable<HttpResponse<{ ddpUserGuid: string }>> {
+        return this.postObservable(`/studies/${studyGuid}/participants`, body);
     }
 
     public getList(): Observable<Array<Participant>> {
```

Comment:
Aware this was already here/not one of your changes, but does this work?
This is being called from `manageParticipansComponent. That URL path does not include the `/study`.
I assume this is the route we are trying to reach?
https://pepper-api-spec-dot-broad-ddp-dev.uc.r.appspot.com/#operation/getGovernedStudyParticipants